rasa_core/policies/ensemble.py

```diff
@@ -65,6 +67,7 @@ def train(self, training_trackers, domain, **kwargs):
             for policy in self.policies:
                 policy.train(training_trackers, domain, **kwargs)
             self.training_trackers = training_trackers
+            self.date_trained = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime())
```

Comment:
I'm not sure about `gmtime()` as it might be confusing. can we not use the machine time? I'd suggest we also keep the timestamp format the same as in NLU metadata
https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/model.py#L116

Reply:
Excellent idea, without your prior knowledge I would not have picked up on this use before